modules/parquet/src/lib/wasm/load-wasm-node.ts

```diff
@@ -2,8 +2,14 @@
 // SPDX-License-Identifier: MIT
 // Copyright (c) vis.gl contributors
 
-import * as wasmNode from 'parquet-wasm/node/arrow1';
+// eslint-disable-next-line import/default
+import initWasm from 'parquet-wasm';
+import * as parquetWasm from 'parquet-wasm';
 
 export async function loadWasm(wasmUrl?: string) {
-  return wasmNode;
+  if (typeof initWasm === 'function') {
+    // @ts-expect-error
+    await initWasm('https://unpkg.com/parquet-wasm@0.6.0-beta.1/esm/arrow1_bg.wasm');
+  }
```

Comment:
Assuming that this file is only called from Node (as suggested by the filename), this line should never be true.